### DISCUSSION TITLE
Proper way to translate entire site

### DISCUSSION THREAD

**Discussion by garanda21** [alexjustesen/speedtest-tracker#331] [Ideas]
I was wondering what is the best approach to translate de entire application, I'm aware that exists the folder `lang/en` with a few messages.

But what about messages within the app like `Notification::make()` title ?

I am willing to translate to Spanish, but I want to check first what you think and what is the best approach !

**Comment by alexjustesen** [comment]
Funny you should mention this, just last night I was looking at services that I could use to track translations. I came across https://weblate.org/en/ and I'm evaluating how to integrate it with a Laravel app.

I know we also have a German translation offer too tracked in this discussion https://github.com/alexjustesen/speedtest-tracker/discussions/210

**Reply by garanda21** [reply]
What a coincidence!!


I know and work with a couple of sites, maybe you wanna check:

This:
[poeditor](https://poeditor.com/)

And this one, is open source:
[traduora](https://github.com/ever-co/ever-traduora)

If you need help, let me know!
